### PR TITLE
Add kubeaudit workflow to CI

### PR DESCRIPTION
**Pull request by miseyu**
## Summary
- Set `kubeaudit` to CI to do security audit for each helm chart

### CODE REVIEW

File: .github/workflows/helm_charts_scalar.yml [hunk 1]
```diff
@@ -45,6 +45,43 @@ jobs:
...
+
+      - name: Add Helm repos
+        run: |
+          helm repo add scalar https://scalar-labs.github.io/helm-charts
+
+      - name: Run kubeaudit
+        run: |
+          # TODO If more charts are supported by kubeaudit, they should be added.
+          # Change to `ls charts` when all charts are supported.
+          chart_dirs=$(envoy)
```
Comment (miseyu) [inline .github/workflows/helm_charts_scalar.yml:79]: First of all, only `envoy-charts` have been audited, so other `charts` will be added as soon as they are audited.

File: .github/workflows/helm_charts_scalar.yml [hunk 2]
```diff
@@ -45,6 +45,43 @@ jobs:
...
+
+      - name: Add Helm repos
+        run: |
+          helm repo add scalar https://scalar-labs.github.io/helm-charts
+
+      - name: Run kubeaudit
+        run: |
+          # TODO If more charts are supported by kubeaudit, they should be added.
+          # Change to `ls charts` when all charts are supported.
+          chart_dirs=(envoy)
```
Comment (miseyu) [inline .github/workflows/helm_charts_scalar.yml:79]: First of all, only `envoy-charts` have been audited, so other `charts` will be added as soon as they are audited.

File: .github/workflows/helm_charts_scalar.yml [hunk 3]
```diff
@@ -45,6 +45,43 @@ jobs:
...
+          for chart_dir in ${chart_dirs}; do
```
Comment (superbrothers) [inline .github/workflows/helm_charts_scalar.yml:80]:
```suggestion
          for chart_dir in "${chart_dirs[@]}"; do
```

**Review by superbrothers — APPROVED** [review]

File: .github/workflows/helm_charts_scalar.yml (hunk 3 above)
Comment (miseyu) [reply .github/workflows/helm_charts_scalar.yml]: I'm sorry, I've been remiss. :bow:

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**